pkg/notes/adaptor/validator/schema.ts

```diff
@@ -102,3 +102,54 @@ export const GetNoteResponseSchema = z.object({
   }),
   // ToDo: add attachment_files, reactions
 });
+
+export const RenoteRequestSchema = z.object({
+  content: z.string().max(3000).openapi({
+    example: 'hello world!',
+    description:
+      'Note content (max 3000 characters/if attachment file exists, allow 0 character)',
+    default: '',
+  }),
+  visibility: z.string().openapi({
```

Comment:
```suggestion
  visibility: z
    .union([z.literal('public'), z.literal('home'), z.literal('followers')])
    .openapi({
```
他の API スキーマの定義では使われていないようですが, `z.union` を利用してみてはどうでしょうか? また [v0 の仕様の記述](https://www.notion.so/pulsate-dev/notes-note_id-renote-a82fd87ec850460183fe015501aaca31?pvs=4) では小文字になっていましたが, 大文字にする予定だったでしょうか?